src/main/java/Duke.java

```diff
@@ -1,10 +1,75 @@
+import java.lang.reflect.Array;
```

Comment:
Can I check if this import is being used? Because Array should be present by default in Java.